test/e2e/framework/error/error.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package error
+
+import (
+	"github.com/onsi/gomega"
+
+	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
+)
+
+// ExpectNoError checks if "err" is set, and if so, fails assertion while logging the error.
+func ExpectNoError(err error, explain ...interface{}) {
+	ExpectNoErrorWithOffset(1, err, explain...)
+}
```

Comment:
the PR doesn't seem to be moving the error utils however, it's only copying them into a new package.
see here:
https://github.com/kubernetes/kubernetes/blob/948ad14f7081a844f6b9072aa93b828e9bf2fcbd/test/e2e/framework/util.go#L2041-L2066

Reply:
`ExpectNoError` function was widely used in ./test directory. To make it easier to review, I want to do this step by step. I would delete the corresponding functions in kubernetes/test/e2e/framework/util.go after everything being done.

Reply:
thanks for the explanation. please update the PR title and description to indicate that this is not a complete move, but rather that an intermediate copy is created and that multiple PRs (steps) will follow.

Reply:
@neolit123 Updated